### ISSUE TITLE
No process crash reason sent to clients

### ISSUE THREAD

**Issue by azoimide** [genomizer/genomizer-server#469]

**Comment by Mintey** [comment]
I assume this is an extra field that is supposed to be sent back in the Get /process command along with everything else?

**Comment by azoimide** [comment]
Yes, it could also contain status for running processes, e.g., "Bowte done, started picard" or smth
